app/src/main/java/com/keylesspalace/tusky/adapter/StatusViewHolder.java

```diff
@@ -73,10 +75,11 @@ protected void setupWithStatus(StatusViewData.Concrete status,
 
     }
 
-    private void setRebloggedByDisplayName(final String name) {
+    private void setRebloggedByDisplayName(final CharSequence name, final StatusViewData.Concrete status) {
         Context context = statusInfo.getContext();
-        String boostedText = context.getString(R.string.status_boosted_format, name);
-        statusInfo.setText(boostedText);
+        CharSequence boostedText = context.getString(R.string.status_boosted_format, name);
```

Comment:
minor: I would maybe push the unicode wrap down to here

Reply:
Good point. Fixed in 221537b.